src/components/HeaderForm.js

```diff
@@ -675,6 +675,15 @@ class HeaderForm extends Component {
       this.state.fileSelectOptions
     );
 
+    const DataPositionFormRowComponent = <DataPositionFormRow
```

Comment:
I would use `camelCase` for this here I think, like other variables we use in the tube map. But this is probably fine.